### PR TITLE
docs: add config example with migration guide

### PR DESCRIPTION
**Pull request by gusinacio**

### CODE REVIEW

**Review by carlosvdr — APPROVED** [review]

File: docs/migration-config/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Migration guide
+
+In case you want to quickly migrate from the old stack into the new stack, you can use the following two configurations to help you to migrate without changing too much fields.
+
+You just need to drop the config, use it in the `--config` args and add some additional environment variables.
+
+Please take a deeper look in the config, but 90% of the time you'll be safe just using it. If you find any issues using this config, feel free to open an issue.
```
Comment (suchapalaver) [inline docs/migration-config/README.md:7]: If you check [a Markdown guide](https://google.github.io/styleguide/docguide/style.html#:~:text=Markdown%20content%20follows%20the%20residual,an%2080%2Dcharacter%20line%20limit.) they say .md is supposed be formatted so that if you used `cat` in a terminal and thus limited to 80 chars per line. Up to you.

File: docs/migration-config/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Migration guide
+
+In case you want to quickly migrate from the old stack into the new stack, you can use the following two configurations to help you to migrate without changing too much fields.
```
Comment (suchapalaver) [inline docs/migration-config/README.md:3]:
```suggestion
If you want to migrate quickly from the old stack to the new stack, you can use the following two configurations to help you to do so without changing too many fields.
```

File: docs/migration-config/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+```
+
+You can also add these environment variables in your new setup:
+
+```
+INDEXER_SUBGRAPHS__ESCROW__QUERY_URL=<escrow subgraph query url>
+INDEXER_SUBGRAPHS__ESCROW__DEPLOYMENT_ID=<escrow deployment id>
+```
+
+We recommend you migrating overtime to be fully in the configuration and don't need to provide multiple environment variables that can stay in plaintext, this will make your work easier.
```
Comment (suchapalaver) [inline docs/migration-config/README.md:28]:
```suggestion
We recommend gradually transitioning to a full configuration setup, so you won’t need to rely on multiple environment variables, especially those stored in plaintext. This will simplify your workflow and improve security.
```

File: docs/migration-config/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+## Booleans inside config
+
+The configuration doesn't accept booleans as strings, so in case you had `INDEXER_SERVICE_SERVE_NETWORK_SUBGRAPH` environment variable, please update to `INDEXER_SERVICE__SERVE_NETWORK_SUBGRAPH` and use `true/false` as values (or you could update directly in the configuration).
+
+Please update the config accordingly. Also, check it out the explanation for each field in [config/minimal-config-example.toml](config/minimal-config-example.toml) and also [config/maximal-config-example.toml](config/maximal-config-example.toml)
```
Comment (suchapalaver) [inline docs/migration-config/README.md:34]:
```suggestion
Please update the config accordingly. Also, check out the explanation for each field in [config/minimal-config-example.toml](config/minimal-config-example.toml) and also [config/maximal-config-example.toml](config/maximal-config-example.toml).
```

File: docs/migration-config/arbitrum-config.toml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+[subgraphs.network]
+# Query URL for the Graph Network subgraph.
+query_url = "${INDEXER_SERVICE_NETWORK_SUBGRAPH_ENDPOINT}"
+deployment_id = "${INDEXER_SERVICE_NETWORK_SUBGRAPH_DEPLOYMENT}"
+
+# [subgraphs.escrow]
+# INDEXER_SUBGRAPHS__ESCROW__QUERY_URL
+# query_url = ""
+# INDEXER_SUBGRAPHS__ESCROW__DEPLOYMENT_ID
+# deployment_id = ""
```
Comment (suchapalaver) [inline docs/migration-config/arbitrum-config.toml:26]: Why is this here like this?

File: docs/migration-config/arbitrum-config.toml [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+# INDEXER_SERVICE__SERVE_NETWORK_SUBGRAPH
+# serve_network_subgraph = false
+# Serve the escrow subgraph on `common.server.host_and_port`/escrow
+# INDEXER_SERVICE__SERVE_ESCROW_SUBGRAPH
+# serve_escrow_subgraph = false
+
+free_query_auth_token = "${INDEXER_SERVICE_FREE_QUERY_AUTH_TOKEN}"
+
+########################################
+# Specific configurations to tap-agent #
```
Comment (suchapalaver) [inline docs/migration-config/arbitrum-config.toml:50]:
```suggestion
# tap-agent configurations #
```

Comment (gusinacio) [reply docs/migration-config/arbitrum-config.toml:50]: This was taken from our default values.

File: docs/migration-config/arbitrum-config.toml (hunk 5 above)
Comment (gusinacio) [reply docs/migration-config/arbitrum-config.toml:26]: These are two missing fields that need to be provided by the user

File: docs/migration-config/arbitrum-config.toml (hunk 6 above)
Comment (suchapalaver) [reply docs/migration-config/arbitrum-config.toml:50]:
```suggestion
# Specific configurations for tap-agent #
```
